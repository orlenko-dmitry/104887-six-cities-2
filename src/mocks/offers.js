const offers = [{
  "id": 1,
  "city": {
    "name": `Amsterdam`,
    "location": {
      "latitude": 52.370216,
      "longitude": 4.895168,
      "zoom": 10
    }
  },
  "previewImage": `img/apartment-01.jpg`,
  "images": [`/img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`],
  "title": `Beautiful & luxurious studio at great location`,
  "isFavorite": true,
  "isPremium": true,
  "rating": 4.8,
  "type": `apartment`,
  "bedrooms": 3,
  "max_adults": 4,
  "price": 120,
  "goods": [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  "host": {
    "id": 3,
    "isPro": true,
    "name": `Angelina`,
    "avatar_url": `img/avatar-angelina.jpg`
  },
  "description": `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  "location": {
    "latitude": 52.35514938496378,
    "longitude": 4.673877537499948,
    "zoom": 8
  }
}, {
  "id": 2,
  "city": {
    "name": `Amsterdam`,
    "location": {
      "latitude": 52.370216,
      "longitude": 4.895168,
      "zoom": 10
    }
  },
  "previewImage": `img/room.jpg`,
  "images": [`/img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`],
  "title": `Wood and stone place`,
  "isFavorite": false,
  "isPremium": false,
  "rating": 4.2,
  "type": `private room`,
  "bedrooms": 1,
  "max_adults": 2,
  "price": 80,
  "goods": [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  "host": {
    "id": 3,
    "isPro": true,
    "name": `Angelina`,
    "avatar_url": `img/avatar-angelina.jpg`
  },
  "description": `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  "location": {
    "latitude": 52.35514938496378,
    "longitude": 4.673877537499948,
    "zoom": 8
  }
}, {
  "id": 3,
  "city": {
    "name": `Amsterdam`,
    "location": {
      "latitude": 52.370216,
      "longitude": 4.895168,
      "zoom": 10
    }
  },
  "previewImage": `img/apartment-02.jpg`,
  "images": [`/img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`],
  "title": `Nice, cozy, warm big bed apartmen`,
  "isFavorite": true,
  "isPremium": false,
  "rating": 4,
  "type": `apartment`,
  "bedrooms": 2,
  "max_adults": 3,
  "price": 70,
  "goods": [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  "host": {
    "id": 3,
    "isPro": true,
    "name": `Angelina`,
    "avatar_url": `img/avatar-angelina.jpg`
  },
  "description": `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  "location": {
    "latitude": 52.35514938496378,
    "longitude": 4.673877537499948,
    "zoom": 8
  }
}, {
  "id": 4,
  "city": {
    "name": `Amsterdam`,
    "location": {
      "latitude": 52.370216,
      "longitude": 4.895168,
      "zoom": 10
    }
  },
  "previewImage": `img/apartment-03.jpg`,
  "images": [`/img/room.jpg`, `img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/apartment-03.jpg`],
  "title": `Beautiful luxurious apartment at great location`,
  "isFavorite": false,
  "isPremium": true,
  "rating": 4.9,
  "type": `apartment`,
  "bedrooms": 3,
  "max_adults": 6,
  "price": 200,
  "goods": [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
  "host": {
    "id": 3,
    "isPro": true,
    "name": `Angelina`,
    "avatar_url": `img/avatar-angelina.jpg`
  },
  "description": `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  "location": {
    "latitude": 52.35514938496378,
    "longitude": 4.673877537499948,
    "zoom": 8
  }
}];

export default offers;
