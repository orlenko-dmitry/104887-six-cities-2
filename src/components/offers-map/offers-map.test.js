import React from 'react';
import renderer from 'react-test-renderer';

import OffersMap from '../offers-map/offers-map.jsx';
import offers from '../../mocks/offers.js';

const city = offers[0].city;


it(`OffersMap renders correctly`, () => {
  const tree = renderer.create(
      <OffersMap offers={offers} selectedCity={city}/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
