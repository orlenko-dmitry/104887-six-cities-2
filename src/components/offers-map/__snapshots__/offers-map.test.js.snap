// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffersMap renders correctly 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
